src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
You can delete these files. When you make a react app you can usually remove half the files it creates. clean your app